### DISCUSSION TITLE
deleted

### DISCUSSION THREAD

**Discussion by vijay-dadhich09** [vercel/next.js#13119] [Help]
deleted

**Comment by kyle-mccarthy** [comment]
It looks like you are behind a firewall that is blocking access. The req for the npm package is being proxied through https://nexus.mgmt-bld.oncp.dev which is responsible for the 403 error.

**Comment by vijay-dadhich09** [comment]
deleted

**Reply by timneutkens** [reply]
This doesn't seem like a Next.js bug nor issue so I've converted the issue to a discussion.

**Comment by vijay-dadhich09** [comment]
Hey @timneutkens Can you please delete this discussion? As no longer needed.
